src/lib/pci_virtio_sock.c

```diff
@@ -652,6 +666,12 @@ static struct pci_vtsock_sock *alloc_sock(struct pci_vtsock_softc *sc)
 	s->local_shutdown = 0;
 	s->peer_shutdown = 0;
 
+	if (0 > asprintf(&s->name, "vsock.%d", id)) {
```

Comment:
Ideally, we would fixup the existing debugging messages to use this name. I can do some of that if you'd like but decided not to go too far down that path as it wasn't necessary (but names were useful in figuring out which mount socket was being configured).